starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1>Apple Pies</h1>
+    </header>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <div class="container">
+      <section class="intro">
+        <p>
+          This was my grandmother's apple pie recipe. I have never seen another
+          one quite like it. It will always be my favorite and has won me
+          several first place prizes in local competitions. I hope it becomes
+          one of your favorites as well!
+        </p>
+        <img class="image" src="./images/recipe-info.png" alt="recipe-info" />
+      </section>
+      <hr />
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+      <section>
+        <h3 class="title">Ingredients</h3>
+        <ul class="ingredients">
+          <li>1 recipe pastry for a 9 inch double crust pie</li>
+          <li>1/2 cup unsalted butter</li>
+          <li>3 tablespoons all-purpose flour</li>
+          <li>1/4 cup water</li>
+          <li>3 tablespoons all-purpose flour</li>
+          <li>1/4 cup water</li>
+          <li>1/2 cup white sugar</li>
+          <li>1/2 cup packed brown sugar</li>
+          <li>8 Granny Smith apples - peeled, cored and sliced</li>
```

Comment:
Here, to make it simple you could have set the `list-style-type: none` in the css and added a "-" at the start of every `li`